src/Models/Payment.php

```diff
@@ -94,7 +92,9 @@ public function generateAmountRefunded()
      */
     public function requestRefund($amount = null, $method = 'Stripe')
     {
-        return $this->refund::create([
+        $refundModel = app('refund');
```

Comment:
You can add `/** @var Model $refundModel */` above and it should be ok.  Per discussion yesterday, this package should add an explicit dependency on the refund package and then use the Refund model directly and then work to move that dependency into a service / model interface later.

Reply:
Thanks